### DISCUSSION TITLE
MET plot

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#31] [Comments_HIG_23_001]
plot the MET for signal, Z+jets and TTbar samples

This will help us to make the 2l2q and 2l2nu channels orthogonal to each other. We may need to add MET cut to our 2l2q channel. It should be inverse of what 2l2nu channel uses, which is MET > 150 GeV.

**Comment by jialin-guo1** [comment]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/1d9cca64-7c9b-4621-ac06-a2dc2db64be9)

this is merged case in SR. I will try to zoom into 0-400GeV

**Comment by jialin-guo1** [comment]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/2850d824-6164-4e2d-be1d-cda431186447)


I updated to zoom into 0-400GeV. seems 150GeV cut is good to suppress 2l2q process

**Reply by ram1123** [reply]
It would be good if you can get this also for the data side-band region.

**Reply by jialin-guo1** [reply]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/e1e24827-2742-4562-a545-a8f592300a94)
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/679200be-f6cb-4dde-a048-c77fb2bae931)
yes.  merged and resolved separately

**Reply by ram1123** [reply]
hmmm, now one of the important things for discussion during next week's meeting is the cut on MET. As the analysis is keep delaying, there are chances that the 2l2nu will be part of the publication. So, we have to add this cut. 

But, let's discuss this during the meeting.

By then, can you also print out the yield of signal and background up to 150 GeV and after 150 GeV?

**Reply by ram1123** [reply]
When you get time, also get this number.

**Reply by jialin-guo1** [reply]
Merged case in Signal region:
Met [0,150] GeV:
Number of events from 0 to 150 GeV in zjet : 1559.754156110275
Number of events from 0 to 150 GeV in ttbar : 55.71187487612942
Number of events from 0 to 150 GeV in zv : 528.1040524098055
Number of events from 0 to 150 GeV in ggh : 1557.7495884875348
Number of events from 0 to 150 GeV in vbf : 2460.1338854067435
Met[150:] GeV:
Number of events from 150 in zjet : 5.471374880415329
Number of events from 150 in ttbar : 14.251489464445871
Number of events from 150 in zv : 2.7642334237825925
Number of events from 150 in ggh : 15.566730262956709
Number of events from 150 in vbf : 43.619050807139864

Resolved case in Signal region:
Met [0,150] GeV:
Number of events from 0 to 150 GeV in zjet : 65244.79717818327
Number of events from 0 to 150 GeV in ttbar : 2741.572111525175
Number of events from 0 to 150 GeV in zv : 2930.520025161601
Number of events from 0 to 150 GeV in ggh : 284.4104590568343
Number of events from 0 to 150 GeV in vbf : 206.27670611986315
Met[150:] GeV:
Number of events from 150 in zjet : 28.46553524473334
Number of events from 150 in ttbar : 155.9158238664995
Number of events from 150 in zv : 1.7141057906507935
Number of events from 150 in ggh : 0.7871597556100813
Number of events from 150 in vbf : 1.5862624608505493

**Reply by ram1123** [reply]
- Also, get this for data.

- And plot the normalize to unity the distribution of MET comparing signal (500, 1500, 3000 GeV) and Z+jet MC.

- Significance with and without MET cut.

**Reply by jialin-guo1** [reply]
**Resolved case in Control region**:
Met [0,150] GeV:
Number of events from 0 to 150 GeV in Data SB : 138945.0
Met [150,Inf] GeV:
Number of events from 150 GeV in Data SB : 500.0

**Merged case in Control region**:
Met [0,150] GeV:
Number of events from 0 to 150 GeV in Data SB : 3058.0
Met [150,Inf] GeV:
Number of events from 150 GeV in Data SB : 42.0

**Reply by jialin-guo1** [reply]
**In signal region,And plot the normalize to unity the distribution of MET comparing signal (500, 1500, 3000 GeV) and Z+jet MC.**
m500 VS Zjet:
resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/1a654b17-f161-4828-b5e0-c139e351f758)
merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/383ae275-06d0-47de-9b19-11a007d874c4)

m1500 vs Zjet:
resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/1fce607d-48c1-4915-b2dd-04462d74b162)

merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/a33927ff-1e59-419a-a996-d09f1cd4764a)

m3000 vs Zjet
resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/fb415fe0-22cc-42d4-90f5-d6b8266058ff)

merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/3bc84aa8-69c2-40cc-8929-34a785fabd90)

**Reply by jialin-guo1** [reply]
**Significance with and without MET cut.**

without MET cut in resolved:
s = n_sig/sqrt(n_bkg+n_sig) = 8.12
with MET cut on 150Gev in resolved:
s = n_sig/sqrt(n_bkg+n_sig) = 8.08

with MET cut in merged:
s = n_sig/sqrt(n_bkg+n_sig) = 21.34
with MET cut on 150Gev in merged:
s = n_sig/sqrt(n_bkg+n_sig) = 21.16


Significance don't change a lot with MET cut on 150GeV
